example/example_util.h

```diff
@@ -17,7 +17,28 @@
 
 #include <string>
 #include <parquet/parquet.h>
+#include <stdio.h>
 
 bool GetFileMetadata(const std::string& path, parquet::FileMetaData* metadata);
 
-#endif
+class InputFile {
```

Comment:
This class is redundant now (it came from pull request #18). It is only used in parquet_reader, though, so it's easy to replace it with LocalFile.